packages/thunar.rb

```diff
@@ -25,6 +26,26 @@ class Thunar < Autotools
   depends_on 'hicolor_icon_theme'
   depends_on 'xdg_base'
   depends_on 'wayland'
+  depends_on 'at_spi2_core' # R
+  depends_on 'cairo' # R
+  depends_on 'harfbuzz' # R
+  depends_on 'gdk_pixbuf' # R
+  depends_on 'glib' # R
+  depends_on 'glibc' # R
+  depends_on 'gtk3' # R
+  depends_on 'libice' # R
+  depends_on 'libsm' # R
+  depends_on 'libx11' # R
+  depends_on 'libxfce4ui' # R
+  depends_on 'libxfce4util' # R
+  depends_on 'pango' # R
+  depends_on 'pcre2' # R
+  depends_on 'xfconf' # R
```

Comment:
Do you want to alphabetize these?

Reply:
And can do it manually now but can we do that with the dependency builder in the future? :grin:

Reply:
Done